controls/radwordsprocessing/formats-and-conversion/html/settings.md

```diff
@@ -69,7 +69,19 @@ __Example 1__ shows how you can create and apply specific import settings.
     provider.ImportSettings = importSettings;
 {{endregion}}
 
+> With Q3 2015 the __UriImageSource__ class has been introduced and it is not necessary to subscribe to the LoadFromUri event when you want to import an image with URI.
 
+## UriImageSource 

```

Comment:
If this section is a subsection of the Import Settings, it should be styled with Heading 3. If not, add a TOC.